doc/about.rst

```diff
@@ -159,9 +159,14 @@ Bibtex entry::
     pages = {108--122},
   }
 
+.. _branding-and-logos:
+
 Branding & Logos
 ================
 
+The scikit-learn brand is subject to the following
+:raw-html:`<a href="https://github.com/scikit-learn/blog/assets/brand_guidelines/2025-02-scikit-learn-brand-guidelines.pdf" target="_blank">terms of use and guidelines</a>`.
```

Comment:
This part is not rendering correctly. I don't know the use of the `raw-html` directive, what would be the expected difference w.r.t.
```suggestion
`terms of use and guidelines
<https://github.com/scikit-learn/blog/assets/brand_guidelines/2025-02-scikit-learn-brand-guidelines.pdf>`_.
```